src/pages/vaults/VaultDepositWithdrawForm.tsx

```diff
@@ -511,6 +530,10 @@ export const VaultDepositWithdrawForm = ({
       </AlertMessage>
     ));
 
+  const canAccountTrade = useAppSelector(calculateCanAccountTrade);
+  const maybeConnectWalletButton = !canAccountTrade ? (
+    <OnboardingTriggerButton size={ButtonSize.Base} />
+  ) : undefined;
```

Comment:
`canAccountTrade` is more than just is account connected right? Should we actually be checking if account is connected?

Reply:
nope, it's just if account is connected

Reply:
wait tyler. i think AccountConnected means if we have a dydx private key available, but i think you're looking for an external wallet connection in this case?